R/ds_upsert.R

```diff
@@ -0,0 +1,31 @@
+#' Upsert data to a datastore
+#'
+#' @export
+#' @param resource_id (string) Resource id that the data is going to be stored
+#'   against.
+#' @param force (logical) Set to \code{TRUE} to edit a read-only resource.
+#'   Default: FALSE
+#' @param method (string) Set to \code{insert}, \code{upsert} or \code{update}
+#'   Default: 'upsert'
+#' @param records (list) The data, eg: \code{[{"dob": "2005", "some_stuff":
```

Comment:
the example data you give here is JSON. can you give an example that is R specific, or at least say what is allowed to be passed in. A data.frame? A list? Both? something else?